docs/apis-tools/camunda-api-rest/specifications/sidebar.js

```diff
@@ -113,7 +119,33 @@ module.exports = [
   },
   {
     type: "category",
-    label: "Process instance",
+    label: "Administration",
+    items: [
+      {
+        type: "doc",
+        id: "apis-tools/camunda-api-rest/specifications/pin-the-zeebe-engines-internal-clock-to-a-specific-time-experimental",
+        label:
+          "Pin the Zeebe engine’s internal clock to a specific time (experimental)",
```

Comment:
blocking, for me to look into (and others if they feel like it): 

I'm not sure why, but this endpoint got added in two different sections: 

<img width="379" alt="image" src="https://github.com/user-attachments/assets/36283581-fc7e-40cc-8c44-2f85821f77f2">

Reply:
The endpoint in question has two tags, I guess that is why. That doesn't make much sense and the author probably simply wasn't aware that this happens in our reference docs.

Reply:
So far, our tags refer to the resource the endpoint belongs to and that would be `Clock` here

Reply:
I will own this, and include this change in a PR upstream.